companies/515156685.json

```diff
@@ -1,20 +1,20 @@
 {
-  "name": "Iguazio Systems",
-  "description": "A platform that automates machine learning pipelines to turn AI projects into tangible business results.",
+  "name": "Iguazio (acquired by McKinsey)",
+  "description": "A platform that enables development, deployment and management of AI and Gen AI projects at scale to generate real business impact in the enterprise.",
   "categoryId": 1,
   "size": "m",
   "logoFile": "logos/iguazio_systems-logo.webp",
   "websiteUrl": "https://www.iguazio.com/",
   "careersUrl": "https://www.iguazio.com/careers/",
   "isHiring": true,
-  "linkedinId": null,
+  "linkedinId": "https://www.linkedin.com/company/iguazio/",
   "crunchbaseId": "iguaz-io",
   "finderId": "iguaz-io",
   "xId": null,
   "addresses": [
     {
       "street": "שנקר אריה",
-      "houseNumber": 5,
+      "houseNumber": 18,
       "city": "הרצליה",
       "lat": 32.1596535,
```

Comment:
if 18 (instead of 5) street changes the lat/lon either keep them blank or update as well.

Reply:
they were correct (checked in goolgl maps), but the addres was not (idk how)